functions/nfts/asset/[[index]].ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-unused-modules */
+import { MetaTagInjector } from '../../components/metaTagInjector'
+import getAsset from '../../utils/getAsset'
+
+export const onRequest: PagesFunction = async ({ params, request, next }) => {
+  const { index } = params
+  const collectionAddress = String(index[0])
+  const tokenId = String(index[1])
```

Comment:
Why do you use `String`? This is pretty non-standard - can you instead do something like `index[0]?.toString()`? Or is it guaranteed to be a string?

Is another alternative to change the filepath of this file to capture the collectionAddress and tokenId?